src/components/propositionalLogic/examples.json

```diff
@@ -1,47 +1,47 @@
 {
   "examples":[
         [
-          ["Wir zeigen: a -> (b -> c) -> (a und b) -> c.", "Dies soll bewiesen werden."],
-          ["Es seien a, b und c Aussagen.", "Allgemeine Vorraussetzung"],
-          ["Angenommen $a ->(b -> c)$.", "Vorraussetzung: Die Prämisse a -> (b -> c) ist wahr."],
-          ["Angenommen ferner es gilt $(a und b)$.", "Vorraussetzung: (a und b) ist wahr."],
-          ["Dann folgt a.", "a folgt, da (a und b) nur dann wahr ist, wenn a wahr ist."],
-          ["Ausserdem folgt b.", "b folgt, da (a und b) nur dann wahr ist, wenn b wahr ist."],
-          ["Damit gilt $(b -> c)$.", "Die Konlusion (b -> c) gilt, da die Prämissen a,sowie (a und b) beide gelten."],
-          ["Ferner folgt c.", "Die Konlusion c gilt, da die Prämissen a, sowie (a und b) beide gelten."],
+          ["Wir zeigen: $a -> (b -> c) -> (a und b) -> c$.", "Dies soll bewiesen werden."],
+          ["Es seien $a, b und c$ Aussagen.", "Allgemeine Vorraussetzung"],
```

Comment:
Here is still a $ around the enumeration. I think in an enumaration it should be seperate since otherwise the "und" will be logical?

Reply:
Yes, the "und" here has other semantics. Good job!